app/index.html

```diff
@@ -5,22 +5,107 @@
+                    <li><a href="#" style="border-top-color: pink" class="col-md-1">BLOG</a></li>
+                    <li><a href="#" style="border-top-color: violet" class="col-md-1">PAGES</a></li>
+                    <li><a href="#" style="border-top-color: purple" class="col-md-1">ELEMENTS</a></li>
+                    <li><a href="#" style="border-top-color: darkblue" class="col-md-1">ECOMMERSE</a></li>
+                    <li><a href="#" style="border-top-color: lightblue" class="col-md-1"> MENU</a></li>
+                </ul>   
+               </div>
+            </div>
+           </div>
+          </div>
+        
+        
+    </div>
+    <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+        
+        <ul class="carousel-inner">
+            <li class="item active">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
+                    <button class="active">SEE FEATURES</button>
+                    <button>PURCHASE NOW</button>
+            </li>
+            <li class="item">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
+                    <button class="active">SEE FEATURES</button>
+                    <button>PURCHASE NOW</button>
+            </li>
+            <li class="item">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
+                    <button class="active">SEE FEATURES</button>
+                    <button>PURCHASE NOW</button>
+            </li>
+            <li class="item">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
+                    <button class="active">SEE FEATURES</button>
+                    <button>PURCHASE NOW</button>
+            </li>
+            <li class="item">
+                <h1 style="color:#01BBAC">Creating a unique look.</h1><br/><h1 style="color:white">Never been easier.</h1>
+                    <button class="active">SEE FEATURES</button>
+                    <button>PURCHASE NOW</button>
+            </li>
+        </ul>
+    
+        <ol class="carousel-indicators">
+            <li data-target="#carousel-example-generic" data-slide-to="0" class="active"> </li>
+            <li data-target="#carousel-example-generic" data-slide-to="1"> </li>
+            <li data-target="#carousel-example-generic" data-slide-to="2"> </li>
+            <li data-target="#carousel-example-generic" data-slide-to="3"> </li>
+            <li data-target="#carousel-example-generic" data-slide-to="4"> </li>
+        </ol>
+        
+        </div>    
+    </div>
+    
+    <div class="container">
+    <div id="stealMonth" class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+       <h2> Steal of <span id="sublinhado">the</span> month</h2> 
```

Comment:
Avoid mixing Portuguese and English. Keeping your code in English is always the best choice